### ISSUE TITLE
PSR1 not reporting or fixing alternative tags

### ISSUE THREAD

**Issue by elplaza** [squizlabs/PHP_CodeSniffer#2415]
In [PSR1 documentation](https://www.php-fig.org/psr/psr-1/) says:

> PHP code MUST use the long <?php ?> tags or the short-echo <?= ?> tags; it MUST NOT use the other tag variations.

I noticed that when running the sniffer (PSR1) against a file with alternative (ASP or script) tags, that it does not find the problem:
```bash
$ ./bin/phpcs -s -vvv --standard=PSR1 testfile.php
```
In my opinion the PSR1 ruleset.xml should include also the Generic.PHP.DisallowAlternativePHPTags:
```xml
<rule ref="Generic.PHP.DisallowAlternativePHPTags"/>
```
I'm wrong?

**Comment by gsherwood** [comment]
The PSR1 standard currently satisfies this by banning short open tags. I suppose it could also ban these alternate tags, but they've been removed in PHP7 so I don't think it is worth the extra processing time to look for them.

**Comment by elplaza** [comment]
I think that the extra processing time is not a valid criterion to decide if a sniff must be included or not (otherwise we remove them all :sweat_smile:).
If the problem is the extra processing time, the solution could be insert in the sniff [line 49](https://github.com/squizlabs/PHP_CodeSniffer/blob/880d257c38982752ec3ff9a9f94b6ad0772a6439/src/Standards/Generic/Sniffs/PHP/DisallowAlternativePHPTagsSniff.php#L49) a code similar to this:
```php
if ($this->phpVersion >= 70000) {
    return [];
}
```

**Comment by jrfnl** [comment]
Just a thought: isn't PHP7 all the more reason to sniff for those tags ? After all, if they're not found & fixed, the code will be broken and could present a security leak (code becoming visible as inline HTML).

Hopefully won't happen very often, but might be critical for large applications with some of those tags in forgotten corners.

**Comment by elplaza** [comment]
@jrfnl you are definitely right :+1:.

I don't know why I thought php7 launched a fatal error, instead it renders all the code :worried:.

**Comment by gsherwood** [comment]
Sniff added.
